File: maui-sample/Models/RegistrationDetails.cs
namespace MauiSample.Models
{
  public class RegistrationDetails
  {
    public string RegistrationResult { get; set; } = string.Empty;
    public int LocationPort { get; set; }
    public int LocationSecurePort { get; set; }
    public int ApiPort { get; set; }
    public int ApiSecurePort { get; set; }
    public int LocationV2Port { get; set; }
    public int LocationV2SecurePort { get; set; }
  }
}

File: android-native-sample/app/src/main/java/com/example/kotlin_sample/models/RegistrationDetails.kt
package com.example.kotlin_sample.models

data class RegistrationDetails(
  val registrationResult: String = "",
  val locationPort: Int = 0,
  val locationSecurePort: Int = 0,
  val apiPort: Int = 0,
  val apiSecurePort: Int = 0,
  val locationV2Port: Int = 0,
  val locationV2SecurePort: Int = 0,
)
